run_all_tests.bat

```diff
@@ -5,6 +5,14 @@ SET CurrentDir=%~dp0
 
 call "%~dp0..\..\..\config_env.bat"
 
+
+@echo off
+REM Run all known tests using the IOC Testing Framework
+
+SET CurrentDir=%~dp0
+
+call "%~dp0..\..\..\config_env.bat"
+
```

Comment:
This block appears twice in the header. I presume the intent wasn't to duplicate it?